openapi.yaml

```diff
@@ -18873,11 +18916,354 @@ paths:
                     readOnly: true
         default:
           $ref: "#/components/responses/ErrorResponse"
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+            https://api.linode.com/v4/object-storage/transfer/
+  /placement/groups:
+    x-linode-cli-command: placement
+    get:
+      parameters:
+        - $ref: '#/components/parameters/pageOffset'
+        - $ref: '#/components/parameters/pageSize'
+      summary: Placement Groups List
+      description: >
+        Returns a paginated list of placement groups you have permission to view.
+      tags:
+        - Placement Groups
+      operationId: getPlacementGroups
+      x-linode-cli-action:
+        - list
+        - ls
```

Comment:
Could these become `groups-list` and `groups-ls` to align with the POST request?